theseus/embodied/kinematics/kinematics_model.py

```diff
@@ -36,7 +36,7 @@ def forward_kinematics(self, robot_pose: RobotModelInput) -> Dict[str, LieGroup]
 
 
 class UrdfRobotModel(KinematicsModel):
-    def __init__(self, urdf_path: str):
+    def __init__(self, urdf_path: str, device: str = None):
```

Comment:
Typing should be `Optional[str]` if the argument defaults to None.
Out of curiosity, what does `device=None` do? Does it use the gpu if available and cpu otherwise?

Reply:
yes, gpu if available and cpu otherwise: https://github.com/facebookresearch/differentiable-robot-model/blob/d7bd1b3b8ef1d6dabe9b68474a622185c510e112/differentiable_robot_model/robot_model.py#L101

Reply:
Fixed typing to use Optional[str] here: https://github.com/facebookresearch/theseus/pull/356/commits/272a4de77113abbae09b5d621662de4eb11cb644

Reply:
Good catch @exhaustin!